.github/workflows/docker-publish.yml

```
@@ -1,91 +1,58 @@
-env:
-  DOCKER_IMAGE_NAME: hahihula/eim-idf-build
-  DOCKER_IMAGE_TAG: latest
+    branches: [ main, master ]
+  workflow_dispatch:  # Allow manual triggering
 
 jobs:
-  build-amd64:
+  docker:
+    name: Build and Push Docker Images
     runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)